internal/upgrader/upgrader.go

```diff
@@ -47,6 +47,9 @@ func Upgrade(api CFClient, brokerName string, parallelUpgrades int, dryRun bool,
 	case len(upgradableInstances) == 0:
 		log.Printf("no instances available to upgrade")
 		return nil
+	case checkUpToDate:
+		log.InitialTotals(totalServiceInstances, len(upgradableInstances))
```

Comment:
thoughts: the `log.InitialTotals` logs this sentence: `l.printf("upgradable instances: %d", upgradableInstances)`

I am not sure if it reflects the reality, I mean, are all of them upgradable?
In any case, I am happy with the modification and it seems reasonable